xc7/fasm2bels/fasm2bels.py

```diff
@@ -271,13 +272,18 @@ def main():
         if len(parts) == 3:
             maybe_add_pip(top, maybe_get_wire, fasm_line.set_feature)
 
-    top.set_iostandard(iostandards)
+    if args.iostandard_defs:
+        with open(args.iostandard_defs) as fp:
+            defs = json.load(fp)
+            top.set_iostandard_defs(defs)
 
     for tile, tile_features in tiles.items():
         process_tile(top, tile, tile_features)
 
     top.make_routes(allow_orphan_sinks=args.allow_orphan_sinks)
 
+    top.prune_unconnected_ports()
```

Comment:
Can you make this optional and driven by a flag?

Reply:
Done.